### DISCUSSION TITLE
How do I prevent an alternate screen from appearing when executing custom cmds?

### DISCUSSION THREAD

**Discussion by dievilz** [gokcehan/lf#1693] [Q&A] [answered]
I don't know if this has been asked before or if it's in the docs. But pretty much the title. Let's use the "cmd delete" example in the lfrc.example.

```bash
cmd delete ${{
    set -f
    printf "$fx\n"
    printf "delete?[y/n]"
    read ans
    [ "$ans" = "y" ] && rm -rf $fx
}}
```

When I run it, it clears the screen, changes to an alternate screen and prompts to delete the file.

Is there a way to prevent this and, let's say, prompt the user inside lf's UI?

**Comment by DusanLesan** [comment, marked as answer]
It is enough to switch from shell prefix `$` to [shell-async](https://github.com/gokcehan/lf/blob/master/doc.md#prefixes) prefix `%` like:
```shell
cmd delete %{{
	set -f
	printf "$fx\n"
	printf "delete?[y/n]"
	read ans
	[ "$ans" = "y" ] && rm -rf $fx
	lf -remote "send $id echo"
}}
``` 
edit: added echo at the end to clear status line

**Reply by dievilz** [reply]
Thank you!!
It worked!! 👌🏻

**Reply by DusanLesan** [reply]
You are welcome. I actually prefer it with `$` so I can see all files listed in a separate screen. It is good to know about different prefixes so you can modify it further, but this command as it is written here is not better than built in command (built in automatically accepts one character and have logic to either show how many files it deletes or exact file if it is single).
